Gems/LandscapeCanvas/Code/Source/Editor/MainWindow.cpp

```diff
@@ -2512,6 +2512,25 @@ namespace LandscapeCanvasEditor
     {
         // See comment above in OnPrefabInstancePropagationBegin
         m_prefabPropagationInProgress = false;
+
+        // After prefab propagation is complete, the entity tied to one of our open
+        // graphs might have been deleted (e.g. if a prefab was created from that entity),
+        // so we need to close that graph to be safe. We need to close them in a separate
```

Comment:
nit: By the wording of this part of the sentence, I thought you were going to close all graphs "to be safe", while it really only does it for entities that have been removed. Maybe it can be clarified?

Reply:
Updated!